Unused variable preventing build

Error produced in log file:

```
11:58:05 /home/jenkins/root/workspace/gh_fedora-smoke/xlators/cluster/ec/src/ec.c: In function ‘ec_handle_readmask’:
11:58:05 /home/jenkins/root/workspace/gh_fedora-smoke/xlators/cluster/ec/src/ec.c:1291:13: error: unused variable ‘dict_data’ [-Werror=unused-variable]
11:58:05  1291 |     data_t *dict_data = NULL;
11:58:05       |             ^~~~~~~~~
11:58:05   CC       ec-helpers.lo
11:58:05 cc1: all warnings being treated as errors
```

--- a/xlators/cluster/ec/src/ec.c
+++ b/xlators/cluster/ec/src/ec.c
@@ -1288,7 +1288,6 @@ ec_handle_readmask(call_frame_t *frame, xlator_t *this, loc_t *loc,
                    dict_t *dict, int32_t flags, dict_t *xdata)
 {
     ec_t *ec = this->private;
-    data_t *dict_data = NULL;
     char *read_mask_str = NULL;
     int32_t op_ret = -1;
     int32_t op_errno = EINVAL;

